sig-aws/README.md

```diff
@@ -37,5 +37,8 @@ Note that the links to display team membership will only work if you are a membe
 | @kubernetes/sig-aws-misc | [link](https://github.com/orgs/kubernetes/teams/sig-aws-misc) | General Discussion |
 
 <!-- BEGIN CUSTOM CONTENT -->
+## Contributing
```

Comment:
So how about a title about the meetings? What I head from what you said is that you want to encourage people to interact on the calls.

Reply:
Will change this as well.